src/main/resources/avro/common.avdl

```diff
@@ -128,9 +130,9 @@ record CigarUnit {
   /**
   `referenceSequence` is only used at mismatches (`SEQUENCE_MISMATCH`)
   and deletions (`DELETE`). Filling this field replaces the MD tag.
-  If the relevant information is not available, leave this field as `null`.
+  If the relevant information is not available, leave this field empty.
   */
-  union { null, string } referenceSequence = null;
+  string referenceSequence;
```

Comment:
Is this the right mapping to proto semantics? Shouldn't it be 
```
string referenceSequence = "";
```
There is a difference I think. In the current version ``referenceSequence`` is optional, and if it's not provided within a JSON object it is defined as null. Here, ``referenceSequence`` is _required_ and must always be specified. 

From the [spec](https://developers.google.com/protocol-buffers/docs/proto3?hl=en#json)

> If a value is missing in the JSON-encoded data or if its value is null, it will be interpreted as the appropriate default value when parsed into a protocol buffer. If a field has the default value in the protocol buffer, it will be omitted in the JSON-encoded data by default to save space. 

So, to me, everywhere we have a ``union { null, string } x = null`` we now need to have ``string x = ""`` to be wire compatible with the proto implementation.